src/styles/mixins/_typography.scss

```diff
@@ -19,10 +19,26 @@
     font-size: 13px;
 }
 
+@mixin fonts-deja-blue {
+    @include font-family;
+    @include font-aliasing;
+
+    font-weight: normal;
+    font-size: 14px;
+}
+
 @mixin common-typography {
     @include fonts;
 
     color: $bdl-gray;
     line-height: $bdl-line-height;
     letter-spacing: .3px;
 }
+
+@mixin common-typography-deja-blue {
+    @include fonts-deja-blue;
+
+    color: $bdl-gray;
+    line-height: $bdl-line-height;
```

Comment:
Is the line height supposed to be the same with the larger font?

Reply:
same line height: 20px

Reply:
I would prefix all these with bdl-

bdl-typography
bdl-commonTypography